### ISSUE TITLE
Issue in the creation of the .IN file for a 3D structural problem

### ISSUE THREAD

**Issue by CassAdri** [gmsafir/gmsafir#45]
Hello, I have a problem with generating a Safir .IN file. My issue comes when I make a structural 3D computation. The beginning of the .IN file generated by gmsh is given in this picture :

![Image1](https://user-images.githubusercontent.com/100410154/204732139-c6892dcf-e6b0-4074-868c-12169ba327e6.png)

When I run this .IN file in Safir (2022.a.3 or 2019), there is an error because some lines are set in a wrong order. These lasts are selected in the picture juste above (DYNAMIC..., NLOAD... and OBLIQUE...). If I move these lines and I correct manually the .IN file like in the picture below :

![Image2](https://user-images.githubusercontent.com/100410154/204732192-6344bd54-14ae-4b34-a0c2-0bce58fffa3b.png)

Safir run correctly. So, can you help me with managing this small "issue" ? In order to avoid the manual correction of the .IN file every time ?

Thanks for advance.

Best regards

**Comment by JMFranssen** [comment]
Dear CassAdri,

I downloaded you GEO and G4S and I tried to open these with the latest version of GmSAFIR, the one which is dated 2022-11-08 and some errors appeared in the log window (that you can open by dragging the mouse up from the lower border of the screen). For example;
Error    : Physical Curve 2 does not exist
Error    : No lax properties yet defined

This seems to indicate that you did not you the same version of GmSAFIR as me when creating your model, hence probably not the latest version..
Could you let us know which date bears your version of GmSAFIR (it appears in the menu window on the left of the screen, just below "Solver/SAFIR")?
If you don't have version 2022-11-08, I suggest you download it and try again.

Regards

**Comment by CassAdri** [comment]
Hello,
thanks a lot for your answer, I have seen that I have the version 2022-07-08- Version 1.0
I will download the latest version and keep you updated with my issue

Thanks again

Regards

**Comment by CassAdri** [comment]
Hello again,
I have downloaded the latest version of GmSAFIR, I have created again the same geometry (a structural beam linked to a 2D beam section). As a reminder, my goal is to compute the thermal deformation encountered by a beam in a fire. The 3 files joined to this message, are the .geo and .g4s file for the beam geometry and the Temperature file distribution in the section.
[BeamModel.geo.txt](https://github.com/gmsafir/gmsafir/files/10130303/BeamModel.geo.txt)
[BeamSection.TEM.txt](https://github.com/gmsafir/gmsafir/files/10130304/BeamSection.TEM.txt)
[BeamModel.g4s.txt](https://github.com/gmsafir/gmsafir/files/10130305/BeamModel.g4s.txt)

And unfortunately I have again the same issue as described in the figures on my first message. Thanks a lot for your time and your help.

Best regards
